src/cli.js

```diff
@@ -84,14 +93,22 @@ const formatReturn = x => {
   }
 }
 
+const cleanup = () => {
```

Comment:
Only used once?

Reply:
It's called twice, once by `handleOut` and once by itself. It needs to be named and have it's own closure, so a var seems like as good a place as any.

Reply:
Word. Wish I could 'resolve' this comment.